docs/disabling_history.rst

```diff
@@ -0,0 +1,77 @@
+Disable Creating Historical Records
+===================================
+
+``save_without_historical_record()`` and ``delete_without_historical_record()``
+-------------------------------------------------------------------------------
+
+These methods are automatically added to a model when registering it for history-tracking
+(i.e. defining a ``HistoricalRecords``  manager on the model),
+and can be called instead of ``save()`` and ``delete()``, respectively.
+
+Using the ``disable_history()`` context manager
+-----------------------------------------------
+
+``disable_history()`` has three ways of being called:
+
+#. With no arguments: This will disable all historical record creation
+   (as if the ``SIMPLE_HISTORY_ENABLED`` setting was set to ``False``; see below)
+   within the context manager's ``with`` block.
+#. With ``only_for_model``: Only disable history creation for the provided model type.
```

Comment:
Perhaps this is better as `models` to match the signature of `isinstance`? This would allow people to disable multiple models in a single context manager and it shouldn't be too much additional complexity on our end.